python/examples/integrations/writing_profiles.ipynb

```diff
@@ -0,0 +1,695 @@
+    "\n",
+    "Here we will check different flavors of writing, and you can check which one of these will meet your current needs. Shall we? "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating simple profiles\n",
+    "\n",
+    "In order for us to get started, let's take a very simple example dataset and profile it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "data = {\n",
+    "    \"col_1\": [1.0, 2.2, 0.1, 1.2],\n",
+    "    \"col_2\": [\"some\", \"text\", \"column\", \"example\"],\n",
+    "    \"col_3\": [4, 2, 3, 5]\n",
+    "}\n",
+    "\n",
+    "df = pd.DataFrame(data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>col_1</th>\n",
+       "      <th>col_2</th>\n",
+       "      <th>col_3</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>1.0</td>\n",
+       "      <td>some</td>\n",
+       "      <td>4</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>2.2</td>\n",
+       "      <td>text</td>\n",
+       "      <td>2</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>0.1</td>\n",
+       "      <td>column</td>\n",
+       "      <td>3</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>1.2</td>\n",
+       "      <td>example</td>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   col_1    col_2  col_3\n",
+       "0    1.0     some      4\n",
+       "1    2.2     text      2\n",
+       "2    0.1   column      3\n",
+       "3    1.2  example      5"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import whylogs as why\n",
+    "\n",
+    "profile = why.log(df)"
```

Comment:
Suggest we name the output here something like "profile_results"

In most cases it is equivalent to the "profile" conceptually, but the reason we have this is so that we can write a collection of profiles at once (if there are more than one profile returned, which currently we only get one, but in the case of segments we might have more).